frontend/assets/icons/viewer/zoom_in.svg.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
```

Comment:
As you are dragging the image in the background, if doing so you pass over the toolbar at the bottom, the inertia is triggered and the drawing starts moving

https://github.com/3drepo/3drepo.io/assets/43547703/00fb14c3-04c9-4410-b36c-ff803722c930

Reply:
That's expected.

Reply:
It works differently for SVGs and PNGs though. For a PNG the dragging continues, but for an SVG the dragging is stopped when you mouse over the toolbar